src/ContainerAppHelper.ts

```diff
@@ -147,6 +147,36 @@ export class ContainerAppHelper {
         }
     }
 
+    /**
+     * Update container app with update and ingress update to avoid failure of acr authentication.
```

Comment:
nit: it would be great to re-word this comment:

_Updates the ingress on an existing Container App. Note: this helps to avoid failure during ACR authentication._

Reply:
Sounds good! I'll have a separate pr to update the comments